x/incentives/types/keys.go

```diff
@@ -0,0 +1,40 @@
+package types
+
+import (
+	"bytes"
+
+	"github.com/ethereum/go-ethereum/common"
+)
+
+// constants
+const (
+	// module name
+	ModuleName = "incentives"
+
+	// StoreKey to be used when creating the KVStore
+	StoreKey = ModuleName
+
+	// RouterKey to be used for message routing
+	RouterKey = ModuleName
+)
+
+// prefix bytes for the EVM persistent store
+const (
+	prefixIncentive = iota + 1
+	prefixGasMeter
+)
+
+// KVStore key prefixes
+var (
+	KeyPrefixIncentive = []byte{prefixIncentive}
+	KeyPrefixGasMeter  = []byte{prefixGasMeter}
+)
+
+func SplitGasMeterKey(key []byte) (contract, userAddr string) {
```

Comment:
@fedekunze Could you check if this is correct? I am not sure how the key in the store exactly looks like. Is it `prefix+contractAddr+userAddr` whereas the `+` is actually an empty string `""`?